applications/nrf_desktop/configuration/nrf52810dmouse_nrf52810/app.overlay

```diff
@@ -2,3 +2,17 @@
 	compatible = "nordic,nrf-spim";
 	status = "okay";
 };
+
+&bt_hci_controller {
+	status = "okay";
+};
+
+&bt_hci_sdc {
+	status = "disabled";
+};
+
+/ {
+	chosen {
+		zephyr,bt-hci = &bt_hci_controller;
+	};
+};
```

Comment:
question: why do we need this in app/samples overlays? Isn't default choice ok?

Reply:
@MarekPieta to comment. I believe it is related to flash size

Reply:
You may consider using snippets for this if required in multiple places

Reply:
Added changes suggested by @nordicjm

Reply:
> @MarekPieta to comment. I believe it is related to flash size

I confirm that it's done to reduce memory consumption (both FLASH and RAM)